### ISSUE TITLE
Deeplabcut-GPU-cluster not use?

### ISSUE THREAD

**Issue by JXXCgit** [DeepLabCut/DeepLabCut#771]
Hi, Dear all, 
i am using the HPC in the university to do my training by using the gpu. i use the 'srun' to go the the 'dash$' and then activate my enviroment, 
[shauyin@curta ~]$ srun --ntasks=4 --time=14:00:00 --mem-per-cpu=4000 --qos=standard --partition=gpu --gres=gpu:2 --pty bash
bash-4.2$ module load Anaconda3/2018.12
bash-4.2$ source activate dlc
(dlc) bash-4.2$ export DLClight=True
(dlc) bash-4.2$  python /home/shauyin/script/dlc-gpu.py 0 ${SLURM_NTASKS}

and then the training start....
i check 'squeue -u shauyin'  to see my occupy with gpu, 

[shauyin@curta ~]$ squeue -u shauyin
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
           4309692       gpu     bash  shauyin  R      29:59      1 g011
it does allocate to gpu.
but i do not know why the training is super slower than using the cpu in the cluster.Can any one figure out what the problem would be. 
and here i also attach my 'dlc-gpu.py' script:
import sys
from os import listdir
import tensorflow as tf 
import deeplabcut 
jobid=int(sys.argv[1])
n_cpus=int(sys.argv[2])
config = tf.ConfigProto(device_count = {'CPU': n_cpus}, inter_op_parallelism_threads=n_cpus, intra_op_parallelism_threads=4)
sess = tf.Session(config=config)
project_path='/home/shauyin/test-shauyin-2020-06-21/'
config_path=project_path + 'config.yaml'
video_path='/scratch/shauyin/originvideo/'
vids=[v for v in listdir(video_path) if '.mp4' in v]
video=vids[jobid]
deeplabcut.train_network(config_path,shuffle=1,trainingsetindex=0,gputouse=1,max_snapshots_to_keep=100,displayiters=100,saveiters=300,maxiters=500000)
deeplabcut.evaluate_network(config_path, plotting=True)
deeplabcut.analyse_videos(config_path, [video_path + video],shuffle=1,gputouse=2,batch_size=8,save_as_csv = True,videotype='.mp4')
deeplabcut.plot_trajectories(config_path,[video_path + video])
deeplabcut.create_labeled_video(config_path,[video_path + video])
deeplabcut.extract_outlier_frames(config_path,[video_path + video])
deeplabcut.refine_labels(config_path)
deeplabcut.merge_datasets(config_path)
print('succeded')

does something wrong inside this script for setting gpu use?

and another questions: i have applied two gpu for use, and how can i use the two gpus in one training? 
and in the command' deeplabcut.train_network(config_path,shuffle=1,trainingsetindex=0,gputouse=1,max_snapshots_to_keep=100,displayiters=100,saveiters=300,maxiters=500000)', 'gputouse=1' , the number '1' here only represent the seqeuence of gpus, or the numeric number? 

thank you for help~
greetings, 
shauyin

**Comment by MMathisLab** [comment]
Gpu rouse is the number that you find in nvidia-smi (for example). Sounds like your gpu is not being used; also it currently cannot run across two GPUs, it needs 1 detonated unit. Otherwise hard to debug your server - hope this helps!
